server/command/auto_respond.go

```diff
@@ -0,0 +1,45 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/config"
+)
+
+func (c *Command) autoRespond(parameters ...string) (string, bool, error) {
+	if len(parameters) == 0 {
+		request := model.OpenDialogRequest{
+			TriggerId: c.Args.TriggerId,
+			URL:       c.Config.PluginURL + config.PathDialog + config.PathAutoRespond,
+			Dialog: model.Dialog{
+				CallbackId: "",
+				Title:      "Auto Respond Message",
+				Elements: []model.DialogElement{
+					{
+						DisplayName: "Autorespond Message",
+						Name:        "auto_respond",
+						Type:        "text",
+						Placeholder: "Enter an autorespond message.",
+					},
+				},
+				SubmitLabel:    "Submit",
+				NotifyOnCancel: false,
+				State:          "",
+			},
+		}
```

Comment:
Here's the `OpenDialogRequest` I mentioned earlier. Should I put anything for `CallbackId`and `State`?

Reply:
For the purposes of this dialog, I think we can leave them blank. @hanzei Do you have any insight on if these need to be used here?